### PR TITLE
Parameterize K8s Deployment's Tolerations in helm chart

### PR DESCRIPTION
**Pull request by jwierzbo**

### CODE REVIEW

**Review by ajanikow — CHANGES_REQUESTED** [review]
Missing doc: https://github.com/arangodb/kube-arangodb/blob/master/chart/kube-arangodb/README.md

File: chart/kube-arangodb/README.md [hunk 1]
```diff
@@ -119,6 +119,33 @@ NodeSelector for Deployment pods.
 
 Default: `{}`
 
+### `operator.tolerations`
+
+Tolerations for Deployment pods.
+
+There is built in configuration (can not be changed):
+```yaml
+Tolerations:
```
Comment (ajanikow) [inline chart/kube-arangodb/README.md:128]: Tolerations -> tolerations

File: chart/kube-arangodb/README.md [hunk 2]
```diff
@@ -119,6 +119,33 @@ NodeSelector for Deployment pods.
...
+  operator: "Exists"
+  effect: "NoExecute"
+  tolerationSeconds: 5
+```
+
+which can be extended by additional entries e.g.:
+```yaml
+tolerations:
+- key: devops
+operator: Exists
```
Comment (ajanikow) [inline chart/kube-arangodb/README.md:143]: Indent?

**Review by ajanikow — CHANGES_REQUESTED** [review]

**Review by informalict — APPROVED** [review]
LGTM

**Review by informalict — APPROVED** [review]
LGTM

**Review by ajanikow — APPROVED** [review]

**Pull request merged**